invenio_rdm_records/services/communities/service.py

```diff
@@ -89,23 +89,25 @@ def _include(self, identity, community_id, record, uow):
             {"community": community_id}
         ).resolve()
 
-        no_comments = {}  # empty `data` to avoid adding comments in the request
+        data = {"payload": {"content": comment, "format": "html"}} if comment else {}
```

Comment:
can we use what is already available in `invenio-requests` instead?
See [here](https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/services/events/service.py#L54) and [here](https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/customizations/event_types.py#L138).

The payload should be:
```
{
    comment: ...,
    format: ...
}